fdbclient/FDBTypes.h

```diff
@@ -1192,4 +1192,24 @@ struct ReadBlobGranuleContext {
 	bool debugNoMaterialize;
 };
 
+// Store metadata associated with each storage server
```

Comment:
nit: "Store metadata associated with each storage server" -> "Storage Server Metadata"

Am not very certain, but may be explaining the usecase for this data-structure by using PW example could be helpful. Thoughts?